Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'slim-rails'
 gem 'bootstrap-sass'
 gem 'font-awesome-sass', '~> 4.5.0'
+gem 'google-webfonts-rails', '~> 0.0.4'
 gem 'animate-scss'
```

Comment:
I don't know if I agree with using this google webfonts gem because it includes a lot of fonts that you don't need.  I would instead just import the individual fonts that you need using a link tag in the application.html.slim 
<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>